galsim/scene.py

```diff
@@ -33,7 +33,8 @@
 
 class COSMOSCatalog(object):
     """
-    A class representing a sample of galaxies from the COSMOS sample with F814W<23.5.
+    A class representing a random subsample of galaxies from the COSMOS sample with F814W<25.2
```

Comment:
Should probably indicate at which galsim version number the default catalog changes, i.e. the meaning of "previous" here.